ldap/servers/slapd/back-ldbm/cache.c

```diff
@@ -2220,8 +2225,16 @@ cache_is_in_cache(struct cache *cache, void *ptr)
         return in_cache;
     }
     bep = (struct backcommon *)ptr;
-    cache_lock(cache);
     in_cache = (bep->ep_state & (ENTRY_STATE_DELETED | ENTRY_STATE_NOTINCACHE)) ? 0 : 1;
-    cache_unlock(cache);
     return in_cache;
```

Comment:
"cache" is no longer needed in this function (unused parameter).  Perhaps we can refactor this function?

Reply:
Ok, I refacteored cache_is_in_cache_nolock in last update.

Reply:
thanks, approval has been set!